src/todo_final.test.ts

```diff
@@ -0,0 +1,140 @@
+import {
+	addTodo,
+	clearAll,
+	removeTodo,
+	completeList,
+	uncompletedList,
+	todoItems,
+} from "./todo_final";
+
+/* Runs before each test */
+beforeEach(() => {
+	clearAll();
+});
+
+///* Runs after each test */
+const originalLog = console.log;
+afterEach(() => {
+	console.log = originalLog;
+});
+
+//test for addtodo
+test("add a todo item to the list of todoItems", () => {
+	addTodo("buy apple", "6/30/21");
+	console.log(todoItems);
+	const length = todoItems.length;
+	expect(length).toEqual(1); //length of one item that is being newly added
+	expect(todoItems[0].text).toEqual("buy apple"); //index[0] since it is a test
+});
+
+//test for remove item
+test("remove an item", () => {
+	todoItems.push(
+		{
+			text: "berry",
+			checked: true,
+			id: 1300,
+			createdAt: "3/20/2021",
+			dueAt: "4/20/2021",
+		},
+
+		{
+			text: "ramisha",
+			checked: false,
+			id: 1400,
+			createdAt: "3/20/2021",
+			dueAt: "4/20/2021",
+		}
+	);
+
+	removeTodo(todoItems[0]); //testing if we can remove the 1st item or  not
+	console.log(todoItems);
+	expect(todoItems.length).toEqual(1);
+	expect(todoItems[0].text).toEqual("berry");
```

Comment:
If the first item in the array ("berry") was removed, shouldn't the new first item have text="ramisha" instead?